indigo_content_api/v1/views.py

```diff
@@ -397,7 +397,7 @@ def determine_place(self):
         super(PublishedDocumentSearchView, self).determine_place()
 
 
-class TaxonomyView(viewsets.ReadOnlyModelViewSet):
+class TaxonomyView(viewsets.ModelViewSet):
```

Comment:
This should definitely be ReadOnly :)

Reply:
must have removed that by mistake, was reverting some changes i made ! :)